tests/marker_test.go

```diff
@@ -34,7 +34,12 @@ var _ = Describe("ovs-cni-marker", func() {
 			if err != nil {
 				panic(fmt.Errorf("%v: %s", err, out))
 			}
-			defer node.RunAtNode("node01", "sudo ovs-vsctl --if-exists del-br br-test")
+			defer func() {
+				_, err = node.RunAtNode("node01", "sudo ovs-vsctl --if-exists del-br br-test")
+				if err != nil {
+					panic(fmt.Errorf("%v: %s", err, out))
```

Comment:
It's possible to use Gomega matchers here ?

Reply:
Is it ok to use matchers in `defer`?